README.md

```diff
@@ -1418,25 +1421,28 @@ meanings we’ll do our best to spell out.
 
 ### Kinds of income
 
-🄳 You must pay federal, state, and in some cases, local **income taxes** on your **income**,
-which is the money you make.
+🄳 **Income taxes** are taxes on **income**, the money you make.
+Generally, U.S. citizens and residents (and some
+[foreigners](https://www.irs.gov/individuals/international-taxpayers/nra-withholding) must
+[file](https://www.irs.gov/publications/p17#en_US_2017_publink1000170407) and pay federal
+taxes as well as state taxes, and, in come cases, local income taxes.
 
-🄳 From a tax perspective, your income is divided between
+🄳 From a tax perspective, income is divided between
 [**ordinary income**](https://www.investopedia.com/terms/o/ordinaryincome.asp), which includes
 wages and salary and interest, and
 [**capital gains**](https://www.investopedia.com/terms/c/capital_gains_tax.asp), the term for
-profit you receive from selling assets, including stocks.
-One key difference between ordinary income and capital gains is you can sell assets for a
-profit or a loss, so this kind of income may be taxed differently, with the ability to
-track both gains and losses.
+profit received from selling assets, including stocks.
```

Comment:
"profit gained by" would actually be clearer; it's not what you receive — it's the gain

(this was a problem with the original)

Reply:
Fixed.